### DISCUSSION TITLE
Is it possible to generate session or token via admin API?

### DISCUSSION THREAD

**Discussion by Ani8819** [ory/kratos#2564] [Q&A] [answered]
I want to generate a session or token using admin API and identity_id (UUID v4).

e.g.
email: ory-user@gmail.com
identity_id: 3ee1dd05-4a35-4bf6-9749-1943ddc3718a 

POST admin API search this identity and generate identity session

**Comment by vinckr** [comment, marked as answer]
Hello @Ani8819 
forgive the late answer 🙏 
How are you running Ory Kratos? 
Can you tell me a bit more about the usecase that prompted this request? 
Feel free to also [open a feature request](https://github.com/ory/kratos/issues/new?assignees=&labels=feat&template=FEATURE-REQUEST.yml) with some more details. 
Is this issue related to your question https://github.com/ory/kratos/issues/2487 ?
